docker-compose.yml

```diff
@@ -0,0 +1,63 @@
+version: '2'
+services:
+  fpm:
+    hostname: fpm.magento2.docker
+    image: shiftedreality/cloud-docker-php:7.1-fpm
+    ports:
+      - 9000
+    links:
+      - db
+    volumes_from:
+      - appdata
+    env_file:
+      - ./docker/global.env
+
+  cli:
+    hostname: cli.magento2.docker
+    image: shiftedreality/cloud-docker-php:7.1-cli
+    links:
+      - db
+    volumes_from:
+      - appdata
+    env_file:
+      - ./docker/global.env
+    environment:
+      - M2SETUP_USE_SAMPLE_DATA=false
+
+  db:
+    image: mariadb:10
+    ports:
+      - 3306
+    volumes_from:
+      - dbdata
+    environment:
+      - MYSQL_ROOT_PASSWORD=magento2
+      - MYSQL_DATABASE=magento2
+      - MYSQL_USER=magento2
+      - MYSQL_PASSWORD=magento2
+
+  web:
+    image: shiftedreality/cloud-docker-nginx:1.9
+    ports:
+      - "8080:80"
+    links:
+      - fpm
+      - db
+    volumes_from:
+      - appdata
+    env_file:
+      - ./docker/global.env
+#    environment:
+#      - VIRTUAL_HOST=magento2.docker
+#      - VIRTUAL_PORT=8080
+#      - HTTPS_METHOD=noredirect
```

Comment:
Did you mean to include commented lines?

Reply:
Yes, this is custom config which require changes to `.hosts` file